lms/djangoapps/instructor/views/api.py

```diff
@@ -3404,19 +3404,19 @@ def update_cohort_assignment(request, course_id):
 @require_global_staff
 @require_http_methods(['POST', 'GET'])
 def cohorts_list_with_assignment(request, course_id):
-    cohort_info = json.loads(cohort_handler(request,course_id).content)
+    cohort_info = json.loads(cohort_handler(request, course_id).content)
     course_id = CourseKey.from_string(course_id)
     cohort_assigments = (
         CourseUserGroup.objects.filter(course_id=course_id).values_list('id', 'cohortassigment__user__email')
     )
-    cohort_assigments_dict = {}
+    cohort_assignments_dict = {}
```

Comment:
I would use `defaultdict` instead of general `dict` object

Reply:
Done